modules/Core/module.php

```diff
@@ -321,7 +321,9 @@ public function __construct(Language $language, Pages $pages, User $user, Naviga
                 'user_id' => $language->get('admin', 'user_id'),
                 'username' => $language->get('user', 'username'),
                 'email_address' => $language->get('user', 'email_address')
-            ]
+            ],
+            false,
+            true
```

Comment:
Why are these now marked as internal and therefore hidden from admins?

Reply:
Those are internal events, Those was discussed in discord to make they internal and aber told to include in this pr

Reply:
I don't think they should be internal. It might be useful for admins to post this type of info to a Discord channel for example for audit purposes, also for use in modules such as Websend to delete a rank from an in game user when their account gets deleted